src/components/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Button = () => {
+  return (
+    <button className='createButton'>
```

Comment:
This component should be reusable, but it's tightly coupled to the create button. See the class and the button text.